src/util/http_utils.js

```diff
@@ -287,6 +287,19 @@ async function parse_request_body(req, options) {
             throw new options.ErrorClass(options.error_invalid_body);
         }
     }
+    if (options.body.type.includes('application/x-www-form-urlencoded')) {
```

Comment:
Why `.includes()` and not just `===`?

Reply:
it can contain more stuff, for instance: ```application/x-www-form-urlencoded; charset=utf-8```